third_party/move/move-compiler-v2/tests/cyclic-instantiation-checker/v1-typing/infinite_instantiations_invalid.exp

```diff
@@ -0,0 +1,139 @@
+
+Diagnostics:
+error: cyclic type instantiation: a cycle of recursive calls causes a type to grow without bound
+  ┌─ tests/cyclic-instantiation-checker/v1-typing/infinite_instantiations_invalid.move:6:16
+  │
+6 │     public fun t<T>() {
+  │                ^
+  │
+  = `t<T>` calls `t<X::Box<T>>` at tests/cyclic-instantiation-checker/v1-typing/infinite_instantiations_invalid.move:6
```

Comment:
This line number is wrong.  The call is at line 7.  Similar problems with other errors below.

File an Urgent bug.